supply-chain-security/compromises/2010/apache.md

```diff
@@ -0,0 +1,11 @@
+# Apache.org incident
+
+Attackers posted a malicious obfuscated URL in JIRA hiding an XSS attack. Once credentials were obtained, attackers modified the JIRA behavior to capture more credentials, and with one of those credentials they logged to the server hosting JIRA, Bugzilla and Confluence for Apache. Most importantly, some of the servers had cached Subversion credentials.
+
+## Impact
+
+Systems had to be reimaged, passwords rotated, the Confluence and Bugzilla databases are considered leaked.
+
+## Type of compromise
+
+Infrastructure compromise.
```

Comment:
@SantiagoTorres I recommend we add `(Attack Chaining)` in [definitions](https://github.com/cncf/sig-security/blob/master/supply-chain-security/compromises/compromise-definitions.md) as a type of compromise because we're seeing more compromises that are made up of multiple kinds of compromises.  Suggest `(Attack Chaining)` for compromise listings of 3 or more be used at the top level chart, indicated here, and list various types of compromise on the actual details page.
For example, this one was application compromise (needs added), Source Code, etc.

Reply:
will do.... #407